src/services/production.ts

```diff
@@ -58,6 +58,9 @@ export const services: ServiceDefinition[] = [
   }],
 
   // KumaAPI
+  [$.mocks, {
+    constant: [],
+  }],
```

Comment:
Do you need to add this if we don't use mocks in production?

Reply:
Rather so that the symbol `TOKENS.mocks` can be resolved in code.

Reply:
ty!